tests/marine/test_marine.py

```diff
@@ -2,85 +2,252 @@
 Note: in order to run the tests, you must put libmarine.so next to the marine_fixtures.py file
 """
 import pytest
+from typing import List, Dict
+from marine import Marine
+
+
+def test_arp_filter_and_parse(
```

Comment:
After turning the tests into a parameterized test, also add tests for FTP and DHCP. The goal is for us to "sign off" on the accuracy of Marine for common protocols. If you have ideas for more protocols, you're more than welcome to add them.